src/index.ts

```diff
@@ -1,59 +1,72 @@
 import { app, BrowserWindow, Menu, MenuItemConstructorOptions } from "electron";
+
+import { IPC_MESSAGE } from "./lib/constants";
+
 declare const MAIN_WINDOW_WEBPACK_ENTRY: any;
 
 // Handle creating/removing shortcuts on Windows when installing/uninstalling.
 if (require("electron-squirrel-startup")) {
   app.quit();
 }
 
-const menuTemplate: MenuItemConstructorOptions[] = [
-  {
-    label: "Game",
-    submenu: [
-      { label: "New", accelerator: "F2", enabled: false },
-      { type: "separator" },
-      { label: "Beginner", type: "checkbox", enabled: false, checked: true },
-      { label: "Intermediate", type: "checkbox", enabled: false },
-      { label: "Expert", type: "checkbox", enabled: false },
-      { label: "Customer...", type: "checkbox", enabled: false },
-      { type: "separator" },
-      { label: "Marks (?)", type: "checkbox", enabled: false, checked: true },
-      { label: "Color", type: "checkbox", enabled: false, checked: true },
-      { label: "Sound", type: "checkbox", enabled: false },
-      { type: "separator" },
-      { label: "Best Times...", enabled: false },
-      { type: "separator" },
-      { label: "Exit", role: "quit" }
-    ]
-  },
-  {
-    label: "Help",
-    submenu: [
-      { label: "Contents", accelerator: "F1", enabled: false },
-      { label: "Search for Help on...", enabled: false },
-      { label: "Using Help", enabled: false },
-      { type: "separator" },
-      { label: "About Minesweeper...", enabled: false }
-    ]
-  }
-];
-
 const isMac = process.platform === "darwin";
 
-if (isMac) {
-  menuTemplate.unshift({ role: "appMenu" });
-}
-
-const menu = Menu.buildFromTemplate(menuTemplate);
-Menu.setApplicationMenu(menu);
-
 const createWindow = () => {
   // Create the browser window.
   const mainWindow = new BrowserWindow({
     height: 600,
+    webPreferences: {
+      nodeIntegration: true
```

Comment:
This ended up being necessary after consuming the `"electron"` module in the renderer process.